### ISSUE TITLE
Napoleon cannot handle class references in arguments

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1396]
I love the idea of napoleon!  But its not 100&#37; working for me.

The issue I have with napoleon is that class references embedded in the docstring are not being properly parsed.  I use numpydoc style documentation,

and a parameter block like

```
#!python

Parameters

----------

Core : :class:`PDSimCore <PDSim.core.core.PDSimCore>` instance or derived class thereof
```

gets rendered to

![Clipboard01.png](https://bitbucket.org/repo/7zgG/images/1023119084-Clipboard01.png)

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1396
- Originally reported by: Ian Bell
- Originally created at: 2014-02-17T13:57:13.392

**Comment by shimizukawa** [comment]
_From Rob Ruana on 2014-03-03 03:20:03+00:00_

Acknowledged, I am looking into this!

**Comment by shimizukawa** [comment]
_From Rob Ruana on 2014-03-09 06:57:16+00:00_

Closes #1396: Param types with inline markup no longer italicized

Napoleon was attempting to render nested inline markup
( [a big no-no!](http://sphinx-doc.org/rest.html) ) for
parameter types. Now, if Napoleon sees any backquotes
in the parameter type, it is rendered without italics.

→ <<cset 18df6421bff82b53d6fed266e8607344b9921df0>>
